gef.py

```diff
@@ -8791,6 +8791,12 @@ def do_invoke(self, argv: List[str]) -> None:
                 addr = int(argv[0], 0)
                 if addr >= entry.page_start and addr < entry.page_end:
                     self.print_entry(entry)
+            elif argv[0][0] in 'r-' and \
```

Comment:
Can you write a test for this?

Can't we do `elif all(argv[0] in "rwx-")`, I think that would be a way to determine that its in the correct format (albeit not the correct order)